Bump selenium-webdriver

oliverguenther opened the pull request:
@machisuji

oliverguenther committed:
Bump selenium-webdriver

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://oliverguenther-bump-selenium-webdriver.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://oliverguenther-bump-selenium-webdriver.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

oliverguenther committed:
Fix selection tests

oliverguenther committed:
Fix specs for newer webdriver

Webdriver changed Firefox click behavior to simulate a more user-like
behavior when clicking on obstructed elements.

https://github.com/seleniumhq/selenium/issues/1202

oliverguenther committed:
Raise required Firefox ESR
